components/camel-netty/src/main/docs/netty-component.adoc

```diff
@@ -614,6 +614,46 @@ This provides support for timeout and other complexities you otherwise would nee
 You can find an example with the Apache Camel source code in the examples directory
 under the `camel-example-netty-custom-correlation` directory.
 
+== Native transport
+
+To enable native transport you need to add additional dependency for epoll or kqueue depending on your OS and CPU arch. To make it easier add the following extension to your `build` section of `pom.xml`:
+
```

Comment:
Mind that the build vs runtime platform is not always the same. So a developer may use a M1 but the production system is on linux x86. 

So part of the ci/cd pipeline or whatever build process you have then take that into account and include the right set of dependency for the target OS platform.

What is in the current doc is for choosing the arch based on current build platform.

Reply:
Ok, what exactly may i add to docs?

It is difficult to provide recommendations for all project setups. It may be standalone app, Spring/Quarkus, Jbang or anything else. Should the docs provide information about each setup?